src/main/java/io/github/thebusybiscuit/slimefun4/core/services/localization/SupportedLanguage.java

```diff
@@ -58,7 +58,7 @@
     MACEDONIAN("mk", false, "a0e0b0b5d87a855466980a101a757bcdb5f77d9f7287889f3efa998ee0472fc0"),
     TAGALOG("tl", true, "9306c0c1ce6a9c61bb42a572c49e6d0ed20e0e6b3d122cc64c339cbf78e9e937");
 
-    public static final SupportedLanguage[] values = values();
+    protected static final SupportedLanguage[] values = values();
```

Comment:
This shouldn't be protected. The point of these values is to replace `values()` and save a ton of time.

Sonar may complain but that's due to it not being able see the use and addon usages. There's a comment to disable Sonar on a line, Google it

Reply:
i found a way to disable it but will this for just me or for everyone else?